src/workspacer.Native/Native/WindowsWindow.cs

```diff
@@ -151,6 +151,10 @@ public bool CanLayout
         public bool IsFocused => Win32.GetForegroundWindow() == _handle;
         public bool IsMinimized => Win32.IsIconic(_handle);
         public bool IsMaximized => Win32.IsZoomed(_handle);
+        public bool IsFullscreen =>
+            (~Win32.GetWindowStyleLongPtr(_handle))
+                .HasFlag(Win32.WS.WS_OVERLAPPED | Win32.WS.WS_BORDER | Win32.WS.WS_THICKFRAME);
```

Comment:
I'm not against inlined methods when it makes sense, but I think this is getting dense to the point that it's hard to read what it actually does.

I'm also curious about the use of the unary bitwise complement operator here.

Wouldn't it be more C# convention to extract the value, and then instead check for the absence of those flags?

Reply:
The main reason for using the bitwise operators was for performance, using a single invert and method call is significantly faster than 3 separate method calls. Also for setting and clearing flags in C# you have to use bitwise operators anyways.